exchange/bidder.go

```diff
@@ -328,6 +337,21 @@ func (bidder *bidderAdapter) doRequest(ctx context.Context, req *adapters.Reques
 	}
 }
 
+func (bidder *bidderAdapter) doTimeoutNotification(timeoutBidder adapters.TimeoutBidder, req *adapters.RequestData) {
+	ctx, cancel := context.WithTimeout(context.Background(), 200*time.Millisecond)
```

Comment:
Should the 200ms timeout be configurable?

Reply:
Pushing down config info is a non-trivial amount of work. Hard coding it at this time won't make that work any harder, and it doesn't seem likely that there will be a need. We can add in another PR as needed.

Reply:
I'm cool with that.